theme.css

```diff
@@ -4148,8 +4149,12 @@ body.anp-card-layout .mod-left-split .workspace-tab-container,
 body.anp-card-layout .mod-right-split .workspace-tab-container {
   padding-left: var(--anp-card-layout-padding, 10px);
   padding-right: var(--anp-card-layout-padding, 10px);
+	padding-bottom: var(--anp-card-layout-padding, 10px);
   background-color: var(--tab-container-background);
 }
+body.anp-card-layout .mod-right-split {
+ padding-right: var(--anp-card-layout-padding, 10px);
+}
```

Comment:
Forgive me, but doesn't [this](https://github.com/AnubisNekhet/AnuPpuccin/blob/01073f95d5d213016ebfbd98a529b827f24df3d3/src/modules/Workspace/Layouts/cards.scss#L92-L99) achieve the same effect?
```css
body.anp-card-layout .mod-left-split .workspace-tab-container, body.anp-card-layout .mod-right-split .workspace-tab-container {
    padding-left: var(--anp-card-layout-padding, 10px);
    padding-right: var(--anp-card-layout-padding, 10px);
    background-color: var(--tab-container-background);
}
```

Reply:
I've noticed that without this additional one, I can't achieve proper spacing on the right. I'll elaborate more in the next message.